test/transport/data_validator/server_test.exs

```diff
@@ -1,8 +1,25 @@
 defmodule Transport.DataValidator.ServerTest do
   use ExUnit.Case, async: false
   alias Transport.DataValidator.Server
+  import Mock
 
   test "publish a message" do
     assert {:ok, _} = Server.validate_data("pouet")
   end
+
+  test "get taskmeta data" do
```

Comment:
I don't see the reason to mock this, as just doing a normal insert in the database before running the test will make it pass.